### PR TITLE
updates

### PR DESCRIPTION
**Pull request by mtungij**

### CODE REVIEW

File: application/controllers/Product.php [hunk 1]
```diff
@@ -65,4 +66,22 @@ public function delete($id)
         $this->session->set_flashdata('delete_product_success', 'Product deleted successfully!');
         redirect('Product/index');
     }
+
+    public function zeroproduct()
```
Comment (alkadoHs) [inline application/controllers/Product.php:70]: Hii ilitakiwa ikae kwenye controller ya report, but achana nayo

Comment (mtungij) [reply application/controllers/Product.php:70]: Ipi

File: application/controllers/SellersReport.php [hunk 2]
```diff
@@ -177,4 +176,42 @@ public function filter_seller_dashboard()
         // var_dump([$orders]);
     }
 
+
+    public function buying_price()
```
Comment (alkadoHs) [inline application/controllers/SellersReport.php:180]: Yes, like this

File: application/views/reports/buying_price.php [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        <a href="<?php echo base_url('sellersReport/print_buyprice') ?>" target="_blank">
+            <button type="button" class="flex items-center justify-center px-4 py-2 text-sm font-medium text-white rounded-lg bg-sky-700 hover:bg-sky-800 focus:ring-4 focus:ring-sky-300 dark:bg-sky-600 dark:hover:bg-sky-700 focus:outline-none dark:focus:ring-sky-800">
+                <svg class="w-4 h-4 mr-2" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" aria-hidden="true">
+                    <path stroke-linecap="round" stroke-linejoin="round" d="M3 16.5v2.25A2.25 2.25 0 005.25 21h13.5A2.25 2.25 0 0021 18.75V16.5m-13.5-9L12 3m0 0l4.5 4.5M12 3v13.5" />
+                </svg>
+                Download Report
+            </button>
+        </a>
+    </div>
+                            </div>
```
Comment (alkadoHs) [inline application/views/reports/buying_price.php:84]: Jitahidi kuzifomati vizuri code, hapa hazijakaa sawa

Comment (mtungij) [reply application/views/reports/buying_price.php:84]: Ok sawa

**Review by alkadoHs — APPROVED** [review]
nice work

File: application/controllers/Product.php (hunk 1 above)
Comment (alkadoHs) [reply application/controllers/Product.php:70]: namaanisha hiyo method ya **zeroproduct**

**Comment by mtungij** [comment]
Sawa

**Pull request merged**